dlt/destinations/impl/destination/configuration.py

```diff
@@ -27,6 +28,19 @@ class CustomDestinationClientConfiguration(DestinationClientConfiguration):
     skip_dlt_columns_and_tables: bool = True
     max_table_nesting: Optional[int] = 0
 
+    def fingerprint(self) -> str:
+        """Returns fingerprint of the custom destination.
+
+        Returns:
+            str: The custom destination fingerprint.
+        """
+        name = (
+            self.destination_callable
+            if isinstance(self.destination_callable, str)
+            else self.destination_callable.__name__
```

Comment:
cool! but IMO a fully qualified function name __module__ + __name__ would be better

Reply:
Done